scripts/ci-build-and-test

```diff
@@ -83,12 +83,15 @@ printf -- "--- remaining installed files ---\n" &&
   find "${PWD}/destdir" &&
   printf -- "---------------------------------\n"
 set -xv
-dir="${INSTALL_PREFIX}"
-while test x"" != x"${dir}"; do
-  rmdir "${PWD}/destdir${dir}" || exit $?
-  dir=`printf -- "${dir}" | sed -e 's|\(.*\)/.*|\1|'`
-done
-rmdir "${PWD}/destdir" || exit $?
+# Skipping the following statements on Windows, because they fail in the mingw64 environment:
```

Comment:
Interesting; why does it fail?  Is there a missing command or non-posix behavior?

Reply:
Actually I don't know. It silently failed, and I didn't see any error messages in the Travis CI logs, unfortunately. Here is a log from a pipeline run where it got past the `${MAKE} install` line but then died before reaching the next `cmake` command:

https://travis-ci.com/github/stukowski/kim-api/builds/199660794

I couldn't tell what caused the failure, mainly because I still got to understand the purpose of that part of the test script. Could you please tell me what it is supposed to do? I will then take another look at it. Perhaps I can get it to work in the MinGW environment.

Reply:
Right!  It is testing that nothing "extra" is being installed by the "make install" command and/or that everything is installed in the expected places.  It's not very complete for this purpose, but it tries to identify gross problems.

From the travis log you point to I see that the installed files have not been deleted by lines 78-80.  Probably this is due to the `INSTALL_PREFIX` variable having the windows drive name in it?

Reply:
If we can't fix this then, I would put the `if` starting at line 76

Reply:
I'll take another look at this later today. [CMake docs](https://cmake.org/cmake/help/latest/envvar/DESTDIR.html) say that the ` make DESTDIR` mechanism shouldn't be used on Windows, like you say (this might not apply to MinGW though). Perhaps it would be better to use the `cmake --install . --prefix` instead. I will do some local testing first before revising the CI script.